lti_redirect/maizey.py

```diff
@@ -14,9 +15,10 @@ def __init__(self, lti_launch_data) -> None:
     def get_restructured_data(self):
       course_title = self.lti_launch_data['https://purl.imsglobal.org/spec/lti/claim/context']['title']
       lis = self.lti_launch_data['https://purl.imsglobal.org/spec/lti/claim/lis']
-
+      expiration_time = datetime.datetime.utcnow() + datetime.timedelta(seconds=300)
```

Comment:
can we make the expiration time configurable, instead of hardcoded?

Reply:
We can?  I am not sure when we would want to adjust it?

Reply:
If you think this is ever going to change then we could leave it as hardcoded.